### PR TITLE
Cdn status

### PR DESCRIPTION
**Pull request by soapy1**

### CODE REVIEW

**Comment by jakirkham** [comment]
Thanks for doing this @soapy1! 😀

File: index.html [hunk 1]
```diff
@@ -78,6 +78,12 @@ <h4 data-l10n-id="systems">Systems</h4>
 
 				</ul>
 
+				<div class="cloning">
+					<h4>CDN cloning status: <span class="status operational">operational</span></h4>
+					<div>For packages scanned in the last 2 weeks see <a href=https://conda-static.anaconda.org/conda-forge/rss.xml>cdn clone rss feed</a></div>
+					<script type="text/javascript" src="cdn-clone.js"></script>
+				</div>
+
```
Comment (scopatz) [inline index.html]: can you also add this blurb to template.html, please?

File: cdn-clone.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        }
+    };
+    xobj.send(null);
+}
+
+function updateStatus(last_updated) {
+  var now_date = new Date();
+  var last_updated_date = new Date(last_updated.trim())
+  var THIRTY_MIN = 30 * 60 * 1000; /* ms */
+  var cloning_status = document.getElementsByClassName("cloning")[0].getElementsByClassName("status")[0]
```
Comment (mariusvniekerk) [inline cdn-clone.js]: `status` feels like a potentially too generic classname which might collide with other things.  Can we rename to `cdn-status`

**Comment by mariusvniekerk** [comment]
Thanks!

**Comment by scopatz** [comment]
Amazing!

**Pull request merged**